modules/ROOT/pages/security/securing-a-graphql-api.adoc

```diff
@@ -0,0 +1,486 @@
+[[securing-an-api]]
+:description: This page is a tutorial on how to secure your API created with the Neo4j GraphQL Library.
+= Securing your GraphQL API
+
+This page is a tutorial on how to secure your API created with the Neo4j GraphQL Library.
+
+
+== Prerequisites
+
+. Set up a new AuraDB instance.
+Refer to link:https://neo4j.com/docs/aura/getting-started/create-instance/[Creating a Neo4j Aura instance].
+. Populate the instance with the Northwind data set.
```

Comment:
Are we sure we want to make this a pre-requisite? This can also be done with an on-prem db